apollo-router/src/plugins/telemetry/tracing/datadog.rs

```diff
@@ -170,13 +196,97 @@ impl TracingConfigurator for Config {
                     .expect("cargo version is set as a resource default;qed")
                     .to_string(),
             )
+            // This prevents spurious errors in the log when talking to the agent
```

Comment:
Another "drive-by", which probably merits some mention in the changelog.

Reply:
Removed and will pull into a separate PR.

Reply:
[0390461](https://github.com/apollographql/router/pull/5609/commits/0390461f5cfa4ffe6e86ca579ab66511211190e4)